### PR TITLE
Newchat

### PR DESCRIPTION
**Pull request by alegomez1**

### CODE REVIEW

File: models/ChatBox.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+
+const chatSchema = new Schema({
```
Comment (adriansdk) [inline models/ChatBox.js]: We already have a chat schema on post model, please remove

File: routes/addPost.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const router = express.Router();
+const Post = require('../models/Post');
+
+
+router.post('/api/post', (req, res, next)=>{
+  Post.create({
```
Comment (adriansdk) [inline routes/addPost.js]: Change this to match our Post model too.

File: views/index.hbs [hunk 3]
```diff
@@ -3,12 +3,22 @@
...
+  <input type="text" name="username"><br>
+  message:<br>
+  <input id="messageInput" type="text" name="message">
+  <button id='messageSubmitButton'>Send Message</button>
+</form>
 
 
-{{!-- Div that the Google Maps will load in --}}
-<div id='map'></div>
+<div id="public-feed">Post So Far</div>
```
Comment (adriansdk) [inline views/index.hbs]: This should be in Beef.js, in the Chat Box column

File: models/ChatBox.js (hunk 1 above)
Comment (AndresMWeber) [reply models/ChatBox.js]: I double checked and chatted with Alex and it seems this isn't a duplicate and is just for his chat system.

**Pull request merged**